src/adapter/src/error.rs

```diff
@@ -98,7 +98,7 @@ pub enum AdapterError {
     /// Transaction cluster was dropped in the middle of a transaction.
     ConcurrentClusterDrop,
     /// Target cluster has no replicas to service query.
-    NoClusterReplicasAvailable(String),
+    NoClusterReplicasAvailable(String, bool),
```

Comment:
```suggestion
    NoClusterReplicasAvailable{
        name: String,
        is_managed: bool,
    },
```

Reply:
Strong +1 on this.

Reply:
Thanks! My Rust is a bit rusty these days 🥁